TestApps/TestAppSwiftUI/AppDelegate.swift

```diff
@@ -29,13 +31,14 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         let appState = application.applicationState
         MobileCore.setLogLevel(.trace)
         MobileCore.configureWith(appId: ENVIRONMENT_FILE_ID)
-        MobileCore.registerExtensions([
-            Identity.self,
-            Edge.self,
-            Assurance.self,
-            EdgeBridge.self,
-            Lifecycle.self
-        ], {
+
+        var extensions: [NSObject.Type] = [Edge.self, Identity.self, EdgeBridge.self, Lifecycle.self]
+
+        #if os(iOS)
+        extensions.append(contentsOf: [Assurance.self])
+        #endif
+
+        MobileCore.registerExtensions(extensions, {
```

Comment:
nit: Can you update the app to use `MobileCore.initialize(appId: ENVIRONMENT_FILE_ID)`? That would replace lines 33-46 and remove the import statement on lines 17-20. You'd also need to update the SceneDelegate by removing the Lifecycle APIs.

Reply:
Thats a good idea!